src/Microsoft.Extensions.Http/Logging/LoggingHttpMessageHandler.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Extensions.Http.Logging
+{
+    public class LoggingHttpMessageHandler : DelegatingHandler
```

Comment:
Use the power of imagination to pretend that there's actually some logging here.

Why two handlers? Well the 'outer' handler makes a scope and will log information about the logical operation. The 'inner' handler (this class) will log when an HTTP request goes over the wire, which may not happen in all cases.